packages/medusa/package.json

```diff
@@ -63,46 +61,36 @@
     "awilix": "^8.0.0",
     "body-parser": "^1.19.0",
     "boxen": "^5.0.1",
-    "bullmq": "^5.4.2",
     "chokidar": "^3.4.2",
-    "class-transformer": "^0.5.1",
-    "class-validator": "^0.14.0",
     "compression": "^1.7.4",
     "connect-redis": "^5.0.0",
     "cookie-parser": "^1.4.6",
     "core-js": "^3.6.5",
     "cors": "^2.8.5",
-    "cross-spawn": "^7.0.3",
     "dotenv": "^16.4.5",
     "express": "^4.18.2",
     "express-session": "^1.17.3",
     "fs-exists-cached": "^1.0.0",
     "glob": "^7.1.6",
     "ioredis": "^5.2.5",
     "ioredis-mock": "8.4.0",
-    "iso8601-duration": "^1.3.0",
     "jsonwebtoken": "^9.0.0",
     "lodash": "^4.17.21",
     "medusa-core-utils": "^1.2.2",
     "medusa-telemetry": "^0.0.17",
     "morgan": "^1.9.1",
     "multer": "^1.4.5-lts.1",
     "node-schedule": "^2.1.1",
-    "papaparse": "5.3.2",
     "passport": "^0.6.0",
     "passport-custom": "^1.1.1",
     "passport-jwt": "^4.0.1",
     "passport-local": "^1.0.0",
     "pg": "^8.11.2",
     "qs": "^6.11.2",
-    "randomatic": "^3.1.1",
     "reflect-metadata": "^0.1.13",
     "regenerator-runtime": "^0.13.11",
     "request-ip": "^3.3.0",
-    "scrypt-kdf": "^2.0.1",
-    "ulid": "^2.3.0",
     "uuid": "^9.0.0",
-    "winston": "^3.8.2",
```

Comment:
Q: aren't we using the logger anymore?

Reply:
@adrien2p winston wasn't used anywhere in the codebase it seems